### PR TITLE
Teodora.Chirila

### PR DESCRIPTION
**Pull request by ChirilaTeodora**
### Pull Request Overview

This pull request adds the documentation for project Air Piano.

### TODO or Help Wanted

This pull request is still a draft . Further modifications of the documentation will be uploaded here.

### Author

Signed-off-by: Chirila Teodora <teochirila11@gmail.com>

### CODE REVIEW

File: docs/project/2024/teodora.chirila/index.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+# Air piano
+
+You can play music just by moving your fingers through air.
+
+Author: Chirila Teodora
+
+GitHub Project Link: https://github.com/UPB-FILS-MA/project-ChirilaTeodora
```
Comment (alexandruradovici) [inline docs/project/2024/teodora.chirila/index.md:7]:
```suggestion

:::info

**Author**: Chirila Teodora \
**GitHub Project Link**: https://github.com/UPB-FILS-MA/project-ChirilaTeodora
```

:::

File: docs/project/2024/teodora.chirila/index.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+Main Component (Software): The Rust program running on the Raspberry Pi Pico W.
+
+Subcomponents:
+
+• Sensor Input: Handles communication with the ultrasonic sensor and retrieves distance data.
+
+• LED Control: Controls the three LEDs based on the detected distance.
+
+• Buzzer Control: Generates tones on the buzzers corresponding to ”played” notes. (Potential addition: Recording functionality to store and playback user input.)
```
Comment (alexandruradovici) [inline docs/project/2024/teodora.chirila/index.md:35]: Please use the markdown syntax for lists.

File: docs/project/2024/teodora.chirila/index.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+• Ultrasonic Sensor: Detects the distance of objects in front of it using ultrasonic waves.
+
+• Three LEDs: Provide visual feedback based on the detected distance.
+
+• Two Buzzers (Active or Passive): Generate sound for the piano functionality.
+
+• Breadboard: Facilitates easy connection between components.
+
+• Jumper Wires: Connect components to the breadboard and Raspberry Pi Pico.
```
Comment (alexandruradovici) [inline docs/project/2024/teodora.chirila/index.md:61]: Please use the markdown syntax for lists.

File: docs/project/2024/teodora.chirila/index.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+### Schematics
+
+![alt text](image.png)
```
Comment (alexandruradovici) [inline docs/project/2024/teodora.chirila/index.md:65]: Please us a title for the image.

File: docs/project/2024/teodora.chirila/index.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+The format is
+
+| [Device](link://to/device) | This is used ... | [price](link://to/store) |
+
+
+
+-->
+
+| Device                                                                                                  | Usage               | Price                                                                                                                                                                                                                                                                                |
+| ------------------------------------------------------------------------------------------------------- | ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
```
Comment (alexandruradovici) [inline docs/project/2024/teodora.chirila/index.md:81]:
```suggestion
|-|-|-|
```

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Review by alexandruradovici — APPROVED** [review]

**Review by ZamfirAndreiPaul — APPROVED** [review]

**Comment by ChirilaTeodora** [comment]
Hello! I just noticed a format issue in the Info section (you can see the screenshot attached). Can I still modify it? 
Thank you!
Title

**Pull request merged**